packages/default/scss/tabstrip/_layout.scss

```diff
@@ -218,12 +233,73 @@ $tabstrip-tab-gap: $nav-item-margin !default;
         }
 
         > .k-content {
-            @include border-left-radius();
+            @include border-left-radius( $tabstrip-item-border-radius );
             // sass-lint:disable no-important
             margin: 0 !important;
             // sass-lint:enable no-important
             border-right-width: 0;
         }
     }
 
+
+    // Selected indicator
+    @if ($tabstrip-indicator-size) {
+
+        .k-tabstrip-items {
+
+            .k-item::after {
+                content: "";
+                border-width: 0;
+                border-style: solid;
+                display: none;
+                position: absolute;
+                z-index: 0;
+                top: 0;
+                right: 0;
+                bottom: 0;
+                left: 0;
+
+                // Variants
+                .k-tabstrip-top & {
+                    border-bottom-width: $tabstrip-indicator-size;
+                }
+                .k-tabstrip-bottom & {
+                    border-top-width: $tabstrip-indicator-size;
+                }
+                .k-tabstrip-left & {
+                    border-right-width: $tabstrip-indicator-size;
+                }
+                .k-tabstrip-right & {
+                    border-left-width: $tabstrip-indicator-size;
+                }
+            }
+            .k-item.k-state-active::after {
+                display: block;
+            }
+
+            .k-item > .k-link {
+                z-index: 1;
+            }
+        }
+
+    }
+
+}
+
+
+
+
```

Comment:
whitespaces :)

Reply:
Yes, whitespaces. It makes code easier to read. I usually leave 4 spaces between major sections of css, namely between `@include exports`  usages.